packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/detail.controller.js

```diff
@@ -0,0 +1,241 @@
+        domain: this.serviceName,
+      })
+      .$promise.then((config) => {
+        this.configuration = config;
+
+        this.configuration.services = filter(
+          map(this.configuration.services, (service, type) =>
+            isObject(service) ? assignIn({ type }, service) : false,
+          ),
+          (service) => isObject(service),
+        );
+        this.configuration.status = {
+          value: this.configuration.status,
+          label: this.configuration.status
+            ? this.$translate.instant('hosted_email_detail_active')
+            : this.$translate.instant('hosted_email_detail_inactive'),
+          icon: this.configuration.status
+            ? 'ovh-font ovh-font-filled-check text-success'
+            : 'ovh-font ovh-font-filled-error text-warning',
+        };
+        return this.configuration;
+      })
+      .catch((err) => {
+        this.TucToast.error(
+          this.$translate.instant('hosted_email_detail_loading_error'),
+        );
+        return this.$q.reject(err);
+      });
+  }
+
+  /**
+   * Get account information
+   * @return {Promise}
+   */
+  getAccount() {
+    return this.OvhApiPackXdslHostedEmail.v6()
+      .getAccount({
+        packId: this.packName,
+        domain: this.serviceName,
+      })
+      .$promise.then((account) => {
+        this.account = account;
+        this.account.offer = {
+          value: this.account.offer,
+          label: this.$translate.instant(
+            `hosted_email_detail_${snakeCase(this.account.offer)}`,
+          ),
+        };
+        this.account.quota = this.toHuman(account.quota.value);
+        this.account.size = this.toHuman(account.size.value);
+        return this.account;
+      })
+      .catch((error) => {
+        this.TucToast.error(
+          this.$translate.instant('hosted_email_detail_loading_error'),
+        );
+        return this.$q.reject(error);
+      });
+  }
+
+  /**
+   * Change the password
+   * @return {Promise}
+   */
+  changePassword() {
+    this.changingPassword = true;
+    return this.OvhApiPackXdslHostedEmail.v6()
+      .changePassword(
+        {
+          packId: this.packName,
+          domain: this.serviceName,
+        },
+        {
+          password: this.password,
+        },
+      )
+      .$promise.then(() => {
+        this.TucToast.success(
+          this.$translate.instant(
+            'hosted_email_detail_change_password_success',
+            { email: this.serviceName },
+          ),
+        );
+        return this.$state.go('telecom.packs.pack');
+      })
+      .catch((error) => {
+        this.TucToast.error(
+          this.$translate.instant('hosted_email_detail_change_password_error'),
+        );
+        return this.$q.reject(error);
+      })
+      .finally(() => {
+        this.changingPassword = false;
+      });
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  getStrength(val) {
+    return (val.length - 8) / 12;
+  }
```

Comment:
Same here, I don't have a clue of what is the purpose.
Also, declare your method as `static`, if no use of `this` is made here. (So you don't need to disable eslint rule)

Reply:
This method is needed by ovh-ng-input-password.

Reply:
Ok, so maybe `getPasswordStrength` name should be good

Reply:
Done